### ISSUE TITLE
Plugins not showing

### ISSUE THREAD

**Issue by pchelferlein** [WWBN/AVideo#3704]
I just have downloaded the latest Version and am running it on a nice small vhost. The Environment is a php 7.4 Apache FPM

**To Reproduce**
1. Go to your site
2. Click on Administration
3. Scroll down to Plugins (http://url/plugins)
4. See no Plugins at all.

**Expected behavior**
To see alist with the default Plugins that come with the Package.

**Error Logs**
Sadly no error was recorded. By opening up the Url https://yoururl/objects/pluginsAvailable.json.php the browser reported a malformed json. 
Passing glue string after array is deprecated. 

The glitch seems to be located in :
/plugin/PlayerSkins/PlayerSkins.php line 24

I changed the original line:

        `return $desc . "<code>" . implode("$names, </code> or <code>") . "</code>";`

To:
        `return $desc . "<code>" . implode("</code> or <code>", $names) . "</code>";`

Then the complete Plugin list reappeared.

**Comment by DanielnetoDotCom** [comment]
Hi, something is wrong in your code, check the original line here https://github.com/WWBN/AVideo/blob/065da9656287f74cdd40fd898d35abf9866987f1/plugin/PlayerSkins/PlayerSkins.php#L24

**Comment by khoiron81** [comment]
Hi All,

Thank you for a great job.
I have the same issue, I just reinstall the 2 days ago, and I want to set up the plugin but loading takes very long, and whenever I activate one plugin it reloads again takes long times to load the plugin page. 

Thanks

**Comment by DanielnetoDotCom** [comment]
@khoiron81 does this update fixes the issue?

**Comment by khoiron81** [comment]
> @khoiron81 does this update fixes the issue?

I change the PlayerSkins.php follow copy paste given code above still the same. Loading takes time

**Comment by DanielnetoDotCom** [comment]
you not suppose to change your code, you suppose to update your site with `git pull`

Please check this https://github.com/WWBN/AVideo/wiki/How-to-Update-your-AVideo-Platform

**Comment by pchelferlein** [comment]
I freshly installed it. I had some issues with an update earlier that is why I was setting it up freshly so i can get it fixxed. I stumbled upon that error by accident.
I switched the logic then to get it fixxed and it worked then. 
I know I am not supposed to tinker in the code around but I thought it might be helpful if I reported it.
Still got a few other minor issues I will report later with some file get contents functions errors in the encoders. 
Cheers
